database/migrations/2024_07_20_125116_create_wallets_table.php

```diff
@@ -15,6 +15,8 @@ public function up(): void
             $table->id();
             $table->string('name');
             $table->enum('type', ['bank', 'cash', 'credit_card', 'mobile']);
+            $table->decimal('balance', 14, 4)->default(0);
+            $table->string('currency')->default('XAF');
```

Comment:
```suggestion
            $table->string('currency')->default('USD');
```

I would use a widely accepted default like `USD`. I am also thinking maybe it should be null?